docs/ref/templates/builtins.txt

```diff
@@ -1115,11 +1115,12 @@ hard-code URLs in your templates::
 
     {% url 'some-url-name' v1 v2 %}
 
-The first argument is a :func:`~django.conf.urls.url` ``name``. It can be a
-quoted literal or any other context variable. Additional arguments are optional
-and should be space-separated values that will be used as arguments in the URL.
-The example above shows passing positional arguments. Alternatively you may
-use keyword syntax::
+The first argument is a :func:`~django.urls.path` (or
```

Comment:
Maybe we should promote the `arg1=v1 arg2=v2` version *before* the `v1 v2` version? i.e. swap ordering.

Also, should we specify how the "v1 v2" is resolved? (based on ordering of pattern, not order of view-arguments)?

Reply:
Maybe, but it's not really related to this patch, correct? Reversing with args works with `path()` as far as I tested.